SharedPackages/BrowserServicesKit/Sources/PixelKit/WideEvent/WideEventData.swift

```diff
@@ -214,16 +218,65 @@ public struct WideEventErrorData: Codable {
     public var code: Int
     public var underlyingDomain: String?
     public var underlyingCode: Int?
+    public var underlyingErrors: [UnderlyingError]
 
     public init(error: Error) {
         let nsError = error as NSError
         self.domain = nsError.domain
         self.code = nsError.code
 
-        if let underlyingError = nsError.userInfo[NSUnderlyingErrorKey] as? NSError {
-            self.underlyingDomain = underlyingError.domain
-            self.underlyingCode = underlyingError.code
+        self.underlyingErrors = Self.collectUnderlyingErrors(from: nsError)
+
+        if let primary = underlyingErrors.first {
+            self.underlyingDomain = primary.domain
+            self.underlyingCode = primary.code
         }
```

Comment:
Not strictly necessary, but nice to have so we don’t have to parse the `underlyingErrors` array.